src/analysis/convex/combination.lean

```diff
@@ -418,9 +418,29 @@ lemma convex_hull_add (s t : set E) : convex_hull R (s + t) = convex_hull R s +
 by simp_rw [←image2_add, ←image_prod, is_linear_map.is_linear_map_add.convex_hull_image,
   convex_hull_prod]
 
+variables (R E)
+/-- `convex_hull` is an additive monoid morphism under pointwise addition. -/
+@[simps]
+def convex_hull_add_monoid_hom : set E →+ set E :=
+{ to_fun := convex_hull R,
+  map_add' := convex_hull_add,
+  map_zero' := convex_hull_zero }
+variables {R E}
+
 lemma convex_hull_sub (s t : set E) : convex_hull R (s - t) = convex_hull R s - convex_hull R t :=
 by simp_rw [sub_eq_add_neg, convex_hull_add, convex_hull_neg]
```

Comment:
Do you want to golf this, for the sake of uniformity?

Reply:
I don't think I can, `map_neg` needs an `add_comm_group` and pointwise subtraction isn't cancellative.